test/pumi.cc

```diff
@@ -11,7 +11,6 @@
 #include <unistd.h>
 #include <iostream>
 #include <algorithm>
-#include <mpi.h>
```

Comment:
If SCOREC_NO_MPI=ON, then unless the user is compiling with the mpi wrappers, then source code and tests cannot `#include <mpi.h>`.  Is that correct?  If so, we should document that in the users and developers guide and that no MPI functions should be called directly within PUMI source.